torch_xla/csrc/tensor.h

```diff
@@ -210,6 +214,29 @@ class XLATensor {
   // Maps from ComputationClient Data unique ID to XLA tensor unique ID.
   using DataUidMap = std::unordered_map<xla::int64, xla::int64>;
 
+  // When a "view" (capture by reference) is taken on a node, an Alias object is
+  // created on the captured node itself, with its current IR Node value.
+  // Inplace operations using the SetIrNode() API to update the current value,
+  // will notice the presence of the alias, and also update the Alias ir_node.
+  struct Alias {
+    explicit Alias(ir::NodePtr ir_node) : ir_node(std::move(ir_node)) {}
```

Comment:
Is the root of `ir_node` ever allowed to be a view? IIUC we're doing "path compression" when taking the view of a view, so maybe add a check that `ir_node` is not a view.

Reply:
A parameter/device-data can have a view, why not?
It's not that we overwrite the value.
We simply track the most current value, and use that when a view()'s user want an IR Node.